bin/runtime/src/lib.rs

```diff
@@ -1033,6 +1033,8 @@ pub type SignedBlock = generic::SignedBlock<Block>;
 /// BlockId type as expected by this runtime.
 pub type BlockId = generic::BlockId<Block>;
 
+pub type Migration = pallet_committee_management::migration::v1::Migration<Runtime>;
```

Comment:
Nit: since we now migrate to `2`, this should be `v2`

Reply:
fixed.